test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -374,6 +374,171 @@ public void RedirectToRoutePermanent_WithParameterRouteName_SetsResultSameRouteN
         }
 
         [Fact]
+        public void Created_WithStringParameter_SetsCreatedLocation()
+        {
+            // Arrange
+            var controller = new Controller();
+            var uri = "http://test/url";
+
+            // Act
+            var result = controller.Created(uri, null);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
+            Assert.Equal(201, result.StatusCode);
+            Assert.Same(uri, result.Location);
+        }
+
+        [Fact]
+        public void Created_WithAbsoluteUriParameter_SetsCreatedLocation()
+        {
+            // Arrange
+            var controller = new Controller();
+            var uri = new Uri("http://test/url");
+
+            // Act
+            var result = controller.Created(uri, null);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
+            Assert.Equal(201, result.StatusCode);
+            Assert.Equal(uri.OriginalString, result.Location);
+        }
+
+        [Fact]
+        public void Created_WithRelativeUriParameter_SetsCreatedLocation()
+        {
+            // Arrange
+            var controller = new Controller();
+            var uri = new Uri("/test/url", UriKind.Relative);
+
+            // Act
+            var result = controller.Created(uri, null);
+
+            // Assert
+            Assert.IsType<CreatedResult>(result);
+            Assert.Equal(201, result.StatusCode);
+            Assert.Equal(uri.OriginalString, result.Location);
+        }
+
+        [Fact]
+        public void CreatedAtAction_WithParameterActionName_SetsResultActionName()
+        {
+            // Arrange
+            var controller = new Controller();
+
+            // Act
+            var result = controller.CreatedAtAction("SampleAction", null);
```

Comment:
@ajaybhargavb  "SampleAction" is used in a lot of places. Isn't it better to define it as a constant?